clar2wasm/src/words/tokens.rs

```diff
@@ -408,13 +415,12 @@ impl ComplexWord for MintNonFungibleToken {
             .i32_const(id_length as i32);
 
         // Push the identifier onto the stack
+        let identifier_ty = generator.nft_types.get(token).cloned().ok_or_else(|| {
+            GeneratorError::TypeError("Usage of nft-burn? on an unknown nft token".to_owned())
```

Comment:
Change `nft-burn?` -> `nft-mint?`.

Reply:
Fixed in [79b8382](https://github.com/stacks-network/clarity-wasm/pull/535/commits/79b8382d6d7b954c45b0180a21fdd11a35d1a93f)